packages/jobs/src/core/Executor.ts

```diff
@@ -65,21 +56,24 @@ export class Executor {
     this.deleteSuccessfulJobs = this.options.deleteSuccessfulJobs
   }
 
+  get jobId() {
+    return `${this.job.path}:${this.job.name}`
+  }
+
   async perform() {
-    this.logger.info(`Started job ${this.job.id}`)
+    this.logger.info(`Started job ${this.jobId}`)
 
-    // TODO break these lines down into individual try/catch blocks?
     try {
-      const job = loadJob({ name: this.job.name, path: this.job.path })
+      const job = await loadJob({ name: this.job.name, path: this.job.path })
       await job.perform(...this.job.args)
 
-      // TODO(@rob): Ask Josh about why this would "have no effect"?
       await this.adapter.success({
         job: this.job,
         deleteJob: DEFAULT_DELETE_SUCCESSFUL_JOBS,
       })
     } catch (error: any) {
-      this.logger.error(`Error in job ${this.job.id}: ${error.message}`)
+      // TODO(jgmw): Handle the error 'any' better
+      this.logger.error(`Error in job ${this.jobId}: ${error.message}`)
```

Comment:
I wanted this to be the job's actual ID so you can find it in the database...it was very common for me to see an error in the logs and then go find the job to get more info.

I do like the idea of having the name/path output in the logs though. I'd probably use a different name than `jobId` because I'd assume that's just the integer ID...`jobName` or `jobPath` or something? Kind of weird that we have `job.name` and `job.path` as well. `fullJobName`, `jobResourceId`, `jobLocation` ...

Reply:
The reason I made this change was because we haven't enforced that the general `BaseJob` must have an ID. That's only available from the prisma job right now. I agree this is less actionable when you see it in the terminal now but I didn't want to go and add the constraint that any job must have a numberic id without discussing it with you.

Reply:
>Kind of weird that we have job.name and job.path as well. 

Not sure I follow what you mean by weird that these are here?

Reply:
> Not sure I follow what you mean by weird that these are here?

haha Sorry, I meant if we added `this.jobName` but there's also `this.job.name` it could be confusing, just like seeing `this.jobId` I assumed would be the same value as `this.job.id`.

Reply:
> The reason I made this change was because we haven't enforced that the general BaseJob must have an ID. That's only available from the prisma job right now. I agree this is less actionable when you see it in the terminal now but I didn't want to go and add the constraint that any job must have a numberic id without discussing it with you.

Good point! I should have added `id` to the required fields in `BaseJob` I think, otherwise there isn't enough info between `name`, `path`, and `args` to uniquely identify it.

I don't think we can force it be only numeric though, some people like uuids and guids for their id fields. GROSS